### PR TITLE
Add freshness monitor notebook

### PR DESCRIPTION
**Pull request by ccjenniferhsu**
Add notebook so that we can create notebook job 

image

### CODE REVIEW

File: freshness_monitoring.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+%pip install "https://ml-team-public-read.s3.us-west-2.amazonaws.com/wheels/data-monitoring/a4050ef7-b183-47a1-a145-e614628e3146/databricks_anomaly_detection-0.0.1-py3-none-any.whl"
+
+# COMMAND ----------
+
+# restart python so library can be installed
+dbutils.library.restartPython()
+
+# COMMAND ----------
+
+dbutils.widgets.text("logging_table_name", "my_logging_table")
```
Comment (adchia) [inline freshness_monitoring.py:12]: want to have this be "catalog.schema.my_logging_table"?

File: freshness_monitoring.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Databricks notebook source
+# install the freshness monitoring wheel
```
Comment (adchia) [inline freshness_monitoring.py:2]: can you also add the legal language in the readme?

File: freshness_monitoring.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+LOGGING_TABLE_NAME = dbutils.widgets.get("logging_table_name")
+CATALOG_NAME = dbutils.widgets.get("catalog_name")
+SCHEMA_NAME =  dbutils.widgets.get("schema_name")
+
+
+# COMMAND ----------
+
+from databricks.data_monitoring.anomalydetection import FreshnessChecker
+
+# Instantiate the freshness checker (add additioanl parameters to instantiation if customization above was performed)
```
Comment (adchia) [inline freshness_monitoring.py:24]: additional*

**Review by adchia — APPROVED** [review]

**Comment by ccjenniferhsu** [comment]
jenkins merge

**Pull request merged**